defaults/manager_versions.yaml

```diff
@@ -0,0 +1,39 @@
+manager_repos_by_branch:
+  master_latest:
+    centos7: 'http://downloads.scylladb.com/manager/rpm/unstable/centos/master/latest/scylla-manager.repo'
+    debian9: 'http://downloads.scylladb.com/manager/deb/unstable/stretch/master/latest/scylladb-manager-master/scylla-manager.list'
+    debian10: 'http://downloads.scylladb.com/manager/deb/unstable/buster/master/latest/scylladb-manager-master/scylla-manager.list'
+    ubuntu16: 'http://downloads.scylladb.com/manager/deb/unstable/xenial/master/latest/scylla-manager-master/scylla-manager.list'
+    ubuntu18: 'http://downloads.scylladb.com/manager/deb/unstable/bionic/master/latest/scylladb-manager-master/scylla-manager.list'
+    ubuntu20: 'http://downloads.scylladb.com/manager/deb/unstable/focal/master/latest/scylla-manager-master/scylla-manager.list'
+  2.4:
+    centos7: 'http://downloads.scylladb.com/rpm/centos/scylladb-manager-2.4.repo'
+    debian9: 'http://downloads.scylladb.com/deb/debian/scylladb-manager-2.4-stretch.list'
+    debian10: 'http://downloads.scylladb.com/deb/debian/scylladb-manager-2.4-buster.list'
+    ubuntu16: 'http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-2.4-xenial.list'
+    ubuntu18: 'http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-2.4-bionic.list'
+    ubuntu20: 'http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-2.4-focal.list'
+  2.3:
+    centos7: 'http://downloads.scylladb.com/rpm/centos/scylladb-manager-2.3.repo'
+    debian9: 'http://downloads.scylladb.com/deb/debian/scylladb-manager-2.3-stretch.list'
+    ubuntu16: 'http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-2.3-xenial.list'
+    ubuntu18: 'http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-2.3-bionic.list'
+  2.2:
+    centos7: 'http://downloads.scylladb.com/rpm/centos/scylladb-manager-2.2.repo'
+    debian9: 'http://downloads.scylladb.com/deb/debian/scylladb-manager-2.2-stretch.list'
+    ubuntu16: 'http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-2.2-xenial.list'
+    ubuntu18: 'http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-2.2-bionic.list'
+
+scylla_backend_repo_by_branch:
+  2020:
+    centos7: 'http://downloads.scylladb.com/rpm/centos/scylla-2020.1.repo'
+    debian9: 'http://downloads.scylladb.com/deb/debian/scylla-2020.1-stretch.list'
+    ubuntu16: 'http://downloads.scylladb.com/deb/ubuntu/scylla-2020.1-xenial.list'
+    ubuntu18: 'http://downloads.scylladb.com/deb/ubuntu/scylla-2020.1-bionic.list'
+  2021:
+    centos7: 'http://downloads.scylladb.com/rpm/centos/scylla-2021.1.repo'
```

Comment:
What about CentOS 8?
Can't we just split it to centos and ubuntu-deb?

Reply:
Since 2020 didn't have unified release, not at the moment